validator/accounts/account.go

```diff
@@ -161,6 +161,16 @@ func CreateValidatorAccount(path string, passphrase string) (string, string, err
 			path = text
 		}
 	}
+	// Forces user to create directory if using non-default path.
+	if path != DefaultValidatorDir() {
+		exists, err := Exists(path)
+		if err != nil {
```

Comment:
Why ignore this error?

Reply:
Good catch, definitely not what was intended.